### DISCUSSION TITLE
Anybody running on Oracle Cloud Ampere A1 (ARM)?

### DISCUSSION THREAD

**Discussion by gingerbeardman** [EasyEngine/easyengine#1679] [Support Requests] [answered]
Currently I run EasyEngine sites on Oracle Cloud AMD instance. Performance is limited.

Does anybody run EasyEngine on Oracle Cloud Ampere A1 (ARM) instance?

I did a quick try but got a docker-related error:

```
Error: There was some error in starting services_global-nginx-proxy_1 container. Please check logs.
```

**Comment by dhsathiya** [comment, marked as answer]
At this moment EE doesn't work on ARM based machines.

**Reply by gingerbeardman** [reply]
Thanks for confirmation!
